pandas/tests/groupby/test_function.py

```diff
@@ -1574,3 +1574,11 @@ def test_corrwith_with_1_axis():
     )
     expected = Series([np.nan] * 6, index=index)
     tm.assert_series_equal(result, expected)
+
+
+def test_multiindex_group_all_columns_when_empty():
```

Comment:
Can you parametrize this over groupby ops: just add `groupby_func` to the arguments and use this to get the method to call.